src/components/X1/X1Dialog.vue

```diff
@@ -1,5 +1,18 @@
 <template>
   <dialog-base :visible.sync="isVisible" class="x1-dialog">
+    <div v-if="soraNetwork !== WALLET_CONSTS.SoraNetwork.Prod" class="disclaimer">
```

Comment:
can be getter in code

get isMainnet(): boolean {
  return this.soraNetwork === WALLET_CONSTS.SoraNetwork.Prod;
}

v-if ! isMainnet